cmd/orb.go

```diff
@@ -236,10 +236,10 @@ func releaseOrb(cmd *cobra.Command, args []string) error {
 		return err
 	}
 
-	response.Orb.Namespace = args[1]
-	response.Orb.Name = args[2]
+	response.Namespace = args[1]
+	response.Name = args[2]
 
-	Logger.Infof("Orb published %s", response.Orb.Ref())
```

Comment:
I thought it was cleaner the way you had it before, since that was paving the way for `api.OrbPublishByID` to return an `*Orb` type instead of an `*OrbPublishRelease` type.

Reply:
@eric-hu I'm happy to make further refactorings in a separate PR.